data.js

```diff
@@ -63,7 +63,17 @@ const emojiItems = [
         musicVideo: "https://www.youtube.com/watch?v=vc6vs-l5dkc",
         genres: ["emo", "pop", "punk"],
         year: 2005
-    },{
+    },
+    {
+        title: "I'll Make a Man Out of You",
+        artist: [" Donny Osmond"],
+        featuredArtist: [],
+        emojiImgs: "9️⃣🕒🌆",
+        musicVideo: "https://www.youtube.com/watch?v=TVcLIfSC4OE",
+        genres: ["Children's"],
```

Comment:
Let's change this to just 'children' so we don't have to worry about the single quotation mark later when we start using these genres for filtering.